Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -23,10 +23,15 @@ import SwiftSyntax
 % for node in [SyntaxBuildableNode(syntax_node) for syntax_node in SYNTAX_NODES if syntax_node.is_buildable()]:
 %   type = node.type()
 %   base_type = node.base_type()
+%   additionalConformance = ""
+%   has_with_trailing_comma_trait = node.node.traits and 'WithTrailingComma' in node.node.traits
+%   if has_with_trailing_comma_trait:
+%     additionalConformance = ", HasTrailingComma"
+%   end
 %   if node.documentation():
 /// ${node.documentation()}
 %   end
-public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_as()} {
+public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_as()}${additionalConformance} {
```

Comment:
You could use the `conformance_clause` helper function to generate the conformance clause here. That way you don’t need to deal with adding the `,` between `ExpressibleAs*` and `HasTrailingComma`.